app/views/devise/confirmations/new.html.erb

```diff
@@ -1,16 +1,20 @@
-<h2>Resend confirmation instructions</h2>
+<div class="container">
+  <div class="row">
+    <div class="col-sm-4 col-sm-offset-4"><h2>Resend confirmation instructions</h2>
```

Comment:
I think it would be better to skip a line between the div opener and the h2 tag. Same goes for line 3 of each single file listed below